services/movie_session.py

```diff
@@ -20,6 +21,11 @@ def get_movies_sessions(session_date: str = None) -> QuerySet:
     return queryset
 
 
+def get_taken_seats(movie_session_id: Any) -> list[dict[str, Any]]:
+    tickets = Ticket.objects.filter(movie_session_id=movie_session_id)
+    return [{"row": ticket.row, "seat": ticket.seat} for ticket in tickets]
+
+
 def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
```

Comment:
It is a good practice to add docstrings to your functions and methods to describe what they do. This will make your code easier to understand and maintain.